docs/concepts/variables.md

```diff
@@ -0,0 +1,90 @@
+---
+description: Prefect variables are dynamic-named, mutable string values, much like environment variables.
+tags:
+    - variables
+    - blocks
+---
+
+# Variables
+
+Variables enable you to store and reuse non-sensitive bits of data, often configuration information. Variables are dynamic-named, mutable string values, much like environment variables. Variables are scoped to a Prefect Server instance or a single workspace in Prefect Cloud.
+
+Variables can be created or modified at any time, but are intended for values with infrequent writes and frequent reads. Variable values may be cached for quicker retrieval.
+
+While variable values are most commonly loaded during flow runtime, they can be loaded in other contexts, at any time, such that they can be used to pass configuration information to Prefect configuration files, such as project steps.
+
+!!! warning "Variables are not Encrypted"
+    Using variables to store sensitive information, such as credentials, is not recommended. Instead, use [Secret blocks](https://docs.prefect.io/concepts/blocks/#prefect-built-in-blocks) to store and access sensitive information.
+
+## Managing variables
+
+You can create, read, edit and delete variables via the Prefect UI, API, and CLI. Names must:
```

Comment:
could be worth saying "much like environment variable naming conventions..." to emphasize the connection?

Reply:
Yup I like that